lib/src/rewrite.rs

```diff
@@ -127,6 +127,24 @@ pub fn rebase_commit(
         .write()?)
 }
 
+pub fn rebase_source(
```

Comment:
Given the name, I would have thought that this was like `jj rebase -s`. Keeping `rebase_to_dest_parent()` seems  better, or maybe there a better name